backend/src/main/java/com/ambacam/model/Passport.java

```diff
@@ -0,0 +1,115 @@
+package com.ambacam.model;
+
+import java.io.Serializable;
+import java.util.Date;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.validation.constraints.NotNull;
+
+@Entity
+public class Passport implements Serializable{
+
+    private static final long serialVersionUID = -7682152333891800618L;
+    
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private Long id;
+    
+    @NotNull(message="the number must not be null")
+    @Column(columnDefinition = "varchar")
+    private String numero;
+    
+    @NotNull(message="the date of issue must not be null")
+    private Date dateDelivrance;
+    
+    private Date dateExpiration;
```

Comment:
Having the expiration date as optional was a conception mistake. Could please make it mandatory and adapt the creation accordingly? The swagger will be modified in accordance.